stable/jenkins/values.yaml

```diff
@@ -249,6 +249,11 @@ Master:
   # This will also overwrite all init scripts
   OverwriteConfig: false
 
+  # By default, the Jobs Map is only used to set the initial jobs the first time
+  # that the chart is installed.  Setting `OverwriteJobs` to `true` will overwrite
+  # the jenkins jobs configuration with the contents of Jobs every time the pod starts.
+  OverwriteJobs: false
```

Comment:
please change to `overwriteJobs`

Reply:
we have started a change of this chart to fit best practices. any help will be appreciated! :)

Reply:
For consistency reasons I chose the capital `O`. Where is this transition to best practice documented?

Anyway, I made the change to `ovewriteJobs` and bumped to the correct chart version.

Reply:
Thanks!
https://helm.sh/docs/chart_best_practices/